### ISSUE TITLE
How to enable the BrowserMobProxy only for certain tests

### ISSUE THREAD

**Issue by ilzul** [serenity-bdd/serenity-core#386]
Hi,

After adding dependency for the serenity-browsermob-plugin into framework, all tests are starting with Proxy. Is there a way to enable the BrowserMobProxy only for certain tests ?

I have tried to add dependency only for net.lightbody.bmp, without any code changes, but the browser is starting with proxy somehow. Could someone explain this magic ?

Thanks

**Comment by YamStranger** [comment]
Hello @ilzul 
Please describe your problem, witch is produced by this behaving. 

Thanks

**Comment by ilzul** [comment]
Hi @YamStranger 

I was able to implement new scenario with BrowserMob Proxy module (this example https://github.com/serenity-bdd/serenity-core/issues/100 was really helful).

But my issue now is that all other UI tests are running with BMP, and I cannot find the proper way how to enable/disable proxy only for certain tests.

I have also tried to add dependency only for net.lightbody.bmp:browsermob-proxy, without implementing the scenario, and I noticed that in this case all tests are still starting with enabled BMP. Seems like ClasspathFixtureProviderService loading those dependencies, and I have no options to manipulate this behavior, am I right ?

Many thanks for your help!
